src/main/java/com/iridium/iridiumskyblock/utils/LocationUtils.java

```diff
@@ -14,6 +14,8 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static com.iridium.iridiumcore.dependencies.xseries.XMaterial.getVersion;
+
```

Comment:
we didn't used static imports anywhere, @PeachesMLG what your thinks about this ?

Reply:
We actually do but it's pretty useless here

Reply:
Yes it is useless for only one, I have to change the import :)